test/run_test.py

```diff
@@ -451,6 +452,9 @@ def run_test(test_module, test_directory, options, launcher_cmd=None, extra_unit
     # If using pytest, replace -f with equivalent -x
     if options.pytest:
         unittest_args = [arg if arg != '-f' else '-x' for arg in unittest_args]
+    elif IS_IN_CI:
+        # use the downloaded test cases configuration, not supported in pytest
+        unittest_args.extend(['--import-slow-tests', '--import-disabled-tests'])
```

Comment:
This is not something I think should be fixed in this PR, but the name unittest_args is slightly misleading as we use it for pytest too

Reply:
there's a larger pytest/unittest initiative, so I will defer this one to that